ilastik/plugins_default/tracking_mamut_export.py

```diff
@@ -111,30 +138,29 @@ def export(self, filename, hypothesesGraph, objectFeaturesSlot, labelImageSlot,
                                 if (np.asarray(features[frame][category][key])).ndim == 2:
                                     for j in range((np.asarray(features[frame][category][key])).shape[1]):
                                         try:
-                                            exists = features[frame][category][key][label, 0]
+                                            _ = features[frame][category][key][label, 0]
                                         except IndexError:
                                             if 'SquaredDistances' in key:
                                                 featureDict[convertKeyName(key) + '_{}'.format(str(j))] = 9999.
```

Comment:
I know it is not part of your changes, but while being at it.... `covertKeyName(key)` is used 5 times in these ~15 lines. Would make a good candidate to save it in a variable.

I also wonder what would happen if `np.asarray(features[frame][category][key])).ndim > 2`? or is that impossible?

Reply:
Thanks for the detailed review! You're helping a lot to get our code clean! 😄  
... I guess you've noticed that this was originally a Python 2 script that I've shaped more and more into a Python 3 plugin here, but now it looks much better :) ...

About the `np.array` dimensionality: this is the return value of vigra's object features. Up to now, they have been at most 2D per object, and AFAIK we have many places where this is implicitly assumed. I added a check nevertheless!